.secrets.baseline

```diff
@@ -128,7 +128,8 @@
     {
       "path": "detect_secrets.filters.regex.should_exclude_file",
       "pattern": [
-        "src/__generated__/.*\\.ts$"
+        "src/__generated__/.*\\.ts$",
+        "^\\.yarn/releases/yarn.*$"
```

Comment:
cc @damassi 

I see you had added some related exclusions when you introduced yarn4 but it appears detect-secrets was still blocking auto upgrades when binary version advanced. So I made this change to ignore the binary all together.